### ISSUE TITLE
Event message should be user friendly if clusterid paraemter is not given in case of remote filesystem 

### ISSUE THREAD

**Issue by Jainbrt** [IBM/ibm-spectrum-scale-csi#220]
**Describe the bug**
Expected reason should be "clusterId must be specified in storageClass for remotecluster"

**To Reproduce**
Created Storage Class with "volBackendFs=remotefs" only while cluster id is  not provided
```
[root@oc-w3 ]# oc describe sc sc-adoring-abhishek-509
Name:                  sc-adoring-abhishek-509
IsDefaultClass:        No
Annotations:           <none>
Provisioner:           spectrumscale.csi.ibm.com
Parameters:            volBackendFs=remotefs
AllowVolumeExpansion:  <unset>
MountOptions:          <none>
ReclaimPolicy:         Delete
VolumeBindingMode:     Immediate
Events:                <none>
```
PVC failed with below reason as expected. It would be good if we can change the event message to have something like this "clusterId must be specified in storageClass for remotecluster"

```
[root@oc-w3 functional-tests]# oc describe  pvc pvc-vigilant-arvind-753
Name:          pvc-vigilant-arvind-753
Namespace:     ibm-spectrum-scale-csi-driver
StorageClass:  sc-adoring-abhishek-509
Status:        Pending
Volume:
Labels:        <none>
Annotations:   volume.beta.kubernetes.io/storage-provisioner: spectrumscale.csi.ibm.com
Finalizers:    [kubernetes.io/pvc-protection]
Capacity:
Access Modes:
VolumeMode:    Filesystem
Mounted By:    <none>
Events:
  Type     Reason                Age                 From                                                                                                 Message
  ----     ------                ----                ----                                                                                                 -------
  Normal   Provisioning          42s (x7 over 106s)  spectrumscale.csi.ibm.com_ibm-spectrum-scale-csi-provisioner-0_978c39ad-be25-452e-bee5-9d3147b4355d  External provisioner is provisioning volume for claim "ibm-spectrum-scale-csi-driver/pvc-vigilant-arvind-753"
  Warning  ProvisioningFailed    42s (x7 over 106s)  spectrumscale.csi.ibm.com_ibm-spectrum-scale-csi-provisioner-0_978c39ad-be25-452e-bee5-9d3147b4355d  failed to provision volume with StorageClass "sc-adoring-abhishek-509": rpc error: code = Internal desc = Unable to check filesystem type of [rgfps0]. Error [Remote call completed with error [400 Bad Request]]
  Normal   ExternalProvisioning  7s (x10 over 106s)  persistentvolume-controller                                                                          waiting for a volume to be created, either by external provisioner "spectrumscale.csi.ibm.com" or manually created by system administrator
```

**Expected behavior**
Expected reason should be "clusterId must be specified in storageClass for remotecluster"

**Environment**
Please run the following an paste your output here:
``` bash
OCP 4.4.4
Scale 5.0.5.0
```

**Comment by Jainbrt** [comment]
Error message on driver pods log :

```
I0602 07:25:08.557919       1 rest_v2.go:44] rest_v2 isStatusOK. statusCode: 400
E0602 07:25:08.557958       1 rest_v2.go:404] Error in list fileset request: Remote call completed with error [400 Bad Request]
I0602 07:25:08.557978       1 rest_v2.go:664] rest_v2 GetFilesystemType. Name: rgfps0
I0602 07:25:08.557992       1 http_utils.go:60] http_utils FormatURL. url: https://oc-w6.ocp.openshift.scale.net:443/
I0602 07:25:08.558024       1 rest_v2.go:560] rest_v2 doHTTP. endpoint: https://oc-w6.ocp.openshift.scale.net:443/scalemgmt/v2/filesystems/rgfps0, method: GET, param: <nil>
I0602 07:25:08.558037       1 http_utils.go:74] http_utils HttpExecuteUserAuth. type: GET, url: https://oc-w6.ocp.openshift.scale.net:443/scalemgmt/v2/filesystems/rgfps0, user: csiadmin
I0602 07:25:08.558345       1 http_utils.go:44] http_utils UnmarshalResponse. response: &{0x6dab00 0xc00030b8c0 0x6e7c60}
I0602 07:25:08.558670       1 rest_v2.go:44] rest_v2 isStatusOK. statusCode: 200
I0602 07:25:08.558831       1 controllerserver.go:442] clusterID not provided in storage Class using Primary ClusterID. Volume Name [pvc-289cd278-1ab1-4aea-a338-aee5bef66143]
I0602 07:25:08.558905       1 controllerserver.go:455] Scale vol create params : &{pvc-289cd278-1ab1-4aea-a338-aee5bef66143 1073741824 rgfps0 true    6544305008369427178 independent  0xc00001ea40 0xc00001ea40 csi-scale-store/.volumes /ibm/gpfs0/csi-scale-store/.volumes gpfs0 /ibm/gpfs0  remotefs}
I0602 07:25:08.558988       1 controllerserver.go:47] reqmap: map[pvc-7a29f66a-2e0c-40b5-ae30-42926346b9ea:1073741824]
I0602 07:25:08.559047       1 controllerserver.go:471] reqmap After: map[pvc-289cd278-1ab1-4aea-a338-aee5bef66143:1073741824 pvc-7a29f66a-2e0c-40b5-ae30-42926346b9ea:1073741824]
I0602 07:25:08.559076       1 rest_v2.go:397] rest_v2 ListFileset. filesystem: rgfps0, fileset: pvc-289cd278-1ab1-4aea-a338-aee5bef66143
I0602 07:25:08.559096       1 http_utils.go:60] http_utils FormatURL. url: https://oc-w6.ocp.openshift.scale.net:443/
I0602 07:25:08.559120       1 rest_v2.go:560] rest_v2 doHTTP. endpoint: https://oc-w6.ocp.openshift.scale.net:443/scalemgmt/v2/filesystems/rgfps0/filesets/pvc-289cd278-1ab1-4aea-a338-aee5bef66143, method: GET, param: <nil>
I0602 07:25:08.559139       1 http_utils.go:74] http_utils HttpExecuteUserAuth. type: GET, url: https://oc-w6.ocp.openshift.scale.net:443/scalemgmt/v2/filesystems/rgfps0/filesets/pvc-289cd278-1ab1-4aea-a338-aee5bef66143, user: csiadmin
I0602 07:25:08.570834       1 http_utils.go:44] http_utils UnmarshalResponse. response: &{0x6dab00 0xc00030a040 0x6e7c60}
I0602 07:25:08.571070       1 rest_v2.go:44] rest_v2 isStatusOK. statusCode: 400
E0602 07:25:08.571096       1 rest_v2.go:671] Unable to get filesystem type for Name rgfps0: Remote call completed with error [400 Bad Request]
E0602 07:25:08.571116       1 utils.go:55] GRPC error: rpc error: code = Internal desc = Unable to check filesystem type of [rgfps0]. Error [Remote call completed with error [400 Bad Request]]
I0602 07:25:08.583022       1 http_utils.go:44] http_utils UnmarshalResponse. response: &{0x6dab00 0xc0001a6140 0x6e7c60}
I0602 07:25:08.583220       1 rest_v2.go:44] rest_v2 isStatusOK. statusCode: 400
E0602 07:25:08.583247       1 rest_v2.go:404] Error in list fileset request: Remote call completed with error [400 Bad Request]
I0602 07:25:08.583265       1 rest_v2.go:664] rest_v2 GetFilesystemType. Name: rgfps0
I0602 07:25:08.583278       1 http_utils.go:60] http_utils FormatURL. url: https://oc-w6.ocp.openshift.scale.net:443/
I0602 07:25:08.583296       1 rest_v2.go:560] rest_v2 doHTTP. endpoint: https://oc-w6.ocp.openshift.scale.net:443/scalemgmt/v2/filesystems/rgfps0, method: GET, param: <nil>
I0602 07:25:08.583308       1 http_utils.go:74] http_utils HttpExecuteUserAuth. type: GET, url: https://oc-w6.ocp.openshift.scale.net:443/scalemgmt/v2/filesystems/rgfps0, user: csiadmin
I0602 07:25:08.601108       1 http_utils.go:44] http_utils UnmarshalResponse. response: &{0x6dab00 0xc00030a1c0 0x6e7c60}
I0602 07:25:08.601345       1 rest_v2.go:44] rest_v2 isStatusOK. statusCode: 400
E0602 07:25:08.601425       1 rest_v2.go:671] Unable to get filesystem type for Name rgfps0: Remote call completed with error [400 Bad Request]
```

**Comment by deeghuge** [comment]
Fixed as part PR#257.
Error message when clusterID is not provided and by default primary cluster id is used for create volume
`volume filesystem fs1 is remotely mounted on Primary cluster, Specify owning cluster ID in storageClass`
